cli/studio.py

```diff
@@ -45,46 +43,67 @@ def scenario_cli():
     "--allow-offset-map",
     is_flag=True,
     default=False,
-    help="Allows Sumo's road network (map.net.xml) to be offset from the origin. if not specified, creates '{}' if necessary.".format(
-        SumoRoadNetwork.shifted_net_file_name
-    ),
+    help="Allows Sumo's road network (map.net.xml) to be offset from the origin. if not specified, creates a network file that ends with AUTOGEN.net.xml if necessary.",
 )
 @click.argument("scenario", type=click.Path(exists=True), metavar="<scenario>")
 def build_scenario(clean, allow_offset_map, scenario):
     _build_single_scenario(clean, allow_offset_map, scenario)
 
 
 def _build_single_scenario(clean, allow_offset_map, scenario):
-    import importlib.resources as pkg_resources
-
-    from smarts.sstudio.sumo2mesh import generate_glb_from_sumo_network
-
     click.echo(f"build-scenario {scenario}")
     if clean:
         _clean(scenario)
 
     scenario_root = Path(scenario)
-    map_net = str(scenario_root / "map.net.xml")
-    if not os.path.isfile(map_net):
+    map_glb = scenario_root / "map.glb"
+    if list(scenario_root.rglob("*.xodr")):
+        from smarts.sstudio.types import MapSpec
+        from smarts.sstudio.od2mesh import generate_glb_from_opendrive_network
+        from smarts.core.opendrive_road_network import OpenDriveRoadNetwork
+
+        map_xodr = str(scenario_root / "map.xodr")
```

Comment:
I have mentioned this before but I think this case presents a specific problem. The glob is looking for `*.xdor` whereas the next step is to assume it is named `map.xdor`.

Reply:
done!